composer.lock

```diff
@@ -8,12 +8,11 @@
     "packages": [
         {
             "name": "automattic/jetpack-asset-tools",
-            "version": "dev-master",
+            "version": "dev-add/protect-module-sync-package",
```

Comment:
we shouldn't merge these changes to the version

Reply:
@lezama - how do i fix this?

Reply:
manually is how I have seen @tyxla  doing it, replacing `dev-add/protect-module-sync-package` with `dev-master`